.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Continuous Integration
+
+on:
+  schedule:
+    - cron: '0 0 1,15 * *' # At 00:00 on day-of-month 1 and 15
+
+env:
+  NODE_OPTIONS: --max-old-space-size=4096
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    needs: [test, lint]
```

Comment:
Cross-workflow job dependency doesn't seem to be supported yet.

see: https://github.community/t/how-do-i-specify-job-dependency-running-in-another-workflow/16482/2

Reply:
Well I'm going to fix it

Reply:
Done